server.js

```diff
@@ -2966,23 +2995,26 @@ cache(function(data, match, sendBadge, request) {
 }));
 
 // GitHub issues integration.
-camp.route(/^\/github\/issues(-raw)?\/([^\/]+)\/([^\/]+)\/?([^\/]+)?\.(svg|png|gif|jpg|json)$/,
+camp.route(/^\/github\/issues(-pr)?(-closed)?(-raw)?\/([^\/]+)\/([^\/]+)\/?([^\/]+)?\.(svg|png|gif|jpg|json)$/,
 cache(function(data, match, sendBadge, request) {
-  var isRaw = !!match[1];
-  var user = match[2];  // eg, badges
-  var repo = match[3];  // eg, shields
-  var ghLabel = match[4];  // eg, website
-  var format = match[5];
-  var apiUrl = 'https://api.github.com/repos/' + user + '/' + repo;
+  var isPR = match[1];
+  var isClosed = match[2];
```

Comment:
Could you explicity convert them to booleans with `!!match[1]` / `!!match[2]`, please?